k8s/krsync

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
```

Comment:
I suggest some hardening of your bash scripts (basically any time you write one). In general we end up with:

```
set -eu
```

which will cause the script to fail if any line errors or if you reference an undefined variable.